go.mod

```diff
@@ -108,8 +109,9 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.12.3
-	istio.io/api v1.20.0-rc.0
-	istio.io/client-go v1.20.0-rc.0
+	istio.io/api v1.20.2-0.20231213020515-8655fab91d5d
+	istio.io/client-go v1.20.2-0.20231213021015-d9f0411d8b67
+	istio.io/pkg v0.0.0-20231221211216-7635388a563e
```

Comment:
Why is this package needed?

Reply:
I don't think I update this intentionally. I will try to revert this changes.